src/client/article.ts

```diff
@@ -71,7 +71,7 @@ function topics(): void {
     const topic = getTopic(follow);
 
     if (topic) {
-        follow?.addEventListener('click', topicClick);
+        follow?.addEventListener('touchstart', topicClick);
```

Comment:
Is this [supported by Safari](https://developer.mozilla.org/en-US/docs/Web/API/Element/touchstart_event#Browser_compatibility)?

Reply:
Tested on my iOS device and it's working. Will make it harder to test on web browsers as there's no touch events, but we're not mocking Bridget anyway at the moment so maybe that's not important.